core/bug_api.php

```diff
@@ -847,6 +853,24 @@ function bug_cache_row( $p_bug_id, $p_trigger_errors = true ) {
 		}
 	}
 
+	if ( is_null( $t_row->token ) ) {
```

Comment:
If we need all issues to have a token, then I would put this into one of two places:

1. When user clicks Share URL, we shouldn't have such URL visible, unless user explicitly requests it.
2. As a database upgrade step.

This doesn't seem to be the right place to update the data.